packages/server/src/Payment/Payment.model.ts

```diff
@@ -0,0 +1,32 @@
+/***
+ * @author Michael Howell <mh.michaelhowell@gmail.com>
+ */
+
+export class Payment {
+
+    constructor(
+        public amount: number,
```

Comment:
Fair point, but part of my reasoning about splitting out the model and schema definitions is so that they can exist in their own forms. By implementing something like this we choose to couple the schema to the model, in this particular example it is fine and works, but it's in other cases where they could be useful I think.

The model and schema should not know about each other, whereas the repository should and converts the two between each other.

You may notice I also removed the toDbo() method from the model because that also couples the schema to the model.